package.json

```diff
@@ -72,9 +72,9 @@
     "npm-run-all": "4.1.2",
     "prop-types": "15.6.1",
     "raw-loader": "0.5.1",
-    "react": "16.2.0",
-    "react-dom": "16.2.0",
-    "react-native": "0.50.3",
+    "react": "16.3.1",
+    "react-dom": "16.3.1",
+    "react-native": "https://github.com/expo/react-native/archive/sdk-28.0.0.tar.gz",
```

Comment:
Why are we adding the packages from the github tar file? isn't it released?

Reply:
it is released, but this is as per https://blog.expo.io/expo-sdk-v28-0-0-is-now-available-f30e8253b530